docs/index.md

```diff
@@ -1,8 +1,154 @@
 ---
-title: Outlines
-template: home.html # Note that this is managed in overrides/home.html
-hide:
-  - navigation
-  - toc
-  - feedback
+title: Welcome to Outlines!
 ---
+
+# Outlines
+
+Outlines is a Python library from [.TXT](https://dottxt.co) that guarantees structured output from 
+large language models. It ensures LLMs speak the language of your application by making them follow specific formats such as JSON, regular expressions, or context-free grammars.
+
+[Get started](/docs/getting_started){ .md-button .md-button--primary }
+[API Reference](/api/reference){ .md-button }
+[Examples](/examples){ .md-button }
+[GitHub](https://github.com/yourusername/outlines){ .md-button }
+
+## Features
+
+<div class="grid cards" markdown>
+
+- :material-shield-check: __Reliable__ - Guaranteed schema compliance -- always valid JSON.
+- :material-puzzle: __Feature-rich__ - Supports a large proportion of the JSON Schema spec, along with regex and context-free grammars.
+- :material-lightning-bolt: __Fast__ - Outlines has negligible runtime overhead, and fast compilation times.
+- :material-cog: __Universal__ - Outlines is a powered by Rust, and can be easily bound to other languages.
```

Comment:
This is more for `outlines-core` documentation imo

Reply:
I'll change this to say that it is extensible through a simple API, since I'd like to keep six totaly cards (and it is also true that Outlines is extensible)